src/overlays/actors/ovl_En_Honotrap/z_en_honotrap.c

```diff
@@ -5,32 +5,70 @@
-void func_8092EDD8(EnHonotrap* this, PlayState* play);
-void func_8092EE1C(EnHonotrap* this, PlayState* play);
-void func_8092EE90(EnHonotrap* this, PlayState* play);
-void func_8092EF98(EnHonotrap* this, PlayState* play);
-void func_8092F074(EnHonotrap* this, PlayState* play);
-void func_8092F0B8(EnHonotrap* this, PlayState* play);
-void func_8092F10C(EnHonotrap* this, PlayState* play);
-void func_8092F208(EnHonotrap* this, PlayState* play);
-void func_8092F3D8(EnHonotrap* this, PlayState* play);
-void func_8092F5EC(EnHonotrap* this, PlayState* play);
-void func_8092F7BC(EnHonotrap* this, PlayState* play);
-void func_8092F878(EnHonotrap* this, PlayState* play);
-
-#if 0
+void EnHonotrap_GetNormal(Vec3f* normal, Vec3f* vec);
+void EnHonotrap_InitEye(EnHonotrap* this, PlayState* play);
+void EnHonotrap_InitFlame(EnHonotrap* this, PlayState* play);
+void EnHonotrap_InitFlameGroup(EnHonotrap* this, PlayState* play);
+void EnHonotrap_SetupEyeIdle(EnHonotrap* this);
+void EnHonotrap_EyeIdle(EnHonotrap* this, PlayState* play);
+void EnHonotrap_SetupEyeOpen(EnHonotrap* this);
+void EnHonotrap_EyeOpen(EnHonotrap* this, PlayState* play);
+void EnHonotrap_SetupEyeAttack(EnHonotrap* this);
+void EnHonotrap_EyeAttack(EnHonotrap* this, PlayState* play);
+void EnHonotrap_SetupEyeClose(EnHonotrap* this);
+void EnHonotrap_EyeClose(EnHonotrap* this, PlayState* play);
+void EnHonotrap_SetupEyeIdle2(EnHonotrap* this);
+void EnHonotrap_EyeIdle2(EnHonotrap* this, PlayState* play);
+void EnHonotrap_SetupEyeOpen2(EnHonotrap* this);
+void EnHonotrap_EyeOpen2(EnHonotrap* this, PlayState* play);
+void EnHonotrap_SetupEyeAttack2(EnHonotrap* this);
+void EnHonotrap_EyeAttack2(EnHonotrap* this, PlayState* play);
+void EnHonotrap_SetupEyeClose2(EnHonotrap* this);
+void EnHonotrap_EyeClose2(EnHonotrap* this, PlayState* play);
+void EnHonotrap_SetupFlame(EnHonotrap* this);
+void EnHonotrap_Flame(EnHonotrap* this, PlayState* play);
+void EnHonotrap_SetupFlameDrop(EnHonotrap* this);
+void EnHonotrap_FlameDrop(EnHonotrap* this, PlayState* play);
+void EnHonotrap_SetupFlameMove(EnHonotrap* this);
+void EnHonotrap_FlameMove(EnHonotrap* this, PlayState* play);
+void EnHonotrap_SetupFlameChase(EnHonotrap* this);
+void EnHonotrap_FlameChase(EnHonotrap* this, PlayState* play);
+void EnHonotrap_SetupFlameVanish(EnHonotrap* this);
+void EnHonotrap_FlameVanish(EnHonotrap* this, PlayState* play);
+void EnHonotrap_SetupFlameGroup(EnHonotrap* this);
+void EnHonotrap_FlameGroup(EnHonotrap* this, PlayState* play);
+void EnHonotrap_UpdateFlame(Actor* thisx, PlayState* play);
+void EnHonotrap_UpdateFlameGroup(Actor* thisx, PlayState* play);
+void EnHonotrap_DrawFlame(Actor* thisx, PlayState* play);
+void EnHonotrap_DrawFlameGroup(Actor* thisx, PlayState* play);
+
+static TexturePtr sSilverEyeTextures[HONOTRAP_EYE_MAX] = { gEyeSwitchSilverOpenTex, gEyeSwitchSilverHalfTex,
+                                                           gEyeSwitchSilverClosedTex, gEyeSwitchSilverClosedTex };
```

Comment:
```suggestion
static TexturePtr sSilverEyeTextures[HONOTRAP_EYE_MAX] = { gEyeSwitchSilverOpenTex, gEyeSwitchSilverHalfTex,
                                                           gEyeSwitchSilverClosedTex, gEyeSwitchSilverClosedTex, };
```
I think this will help with formatting